lib/trailblazer/rails/railtie/loader.rb

```diff
@@ -39,7 +39,14 @@ def self.load_for(app)
         initializer "trailblazer.install", after: "reform.form_extensions" do |app|
           # the trb autoloading has to be run after initializers have been loaded, so we can tweak inclusion of features in
           # initializers.
-          if config.trailblazer.use_loader
+
+          # TODO: remove me in the next version!
+          if config.trailblazer.use_loader.to_s.present?
```

Comment:
For users that maybe have `config.user_loader = true` just checking that the config flag is used not the actual value

Reply:
Why the `to_s`?